lib/modules/media/media_info_view.dart

```diff
@@ -263,6 +271,26 @@ class _PlainScrollCards extends StatelessWidget {
   }
 }
 
+class _ExternalLinkScrollCards extends StatelessWidget {
+  const _ExternalLinkScrollCards({
```

Comment:
In this case, there's no need of a named parameter and we can put it on one line `(this.items)`